.github/workflows/pkg-pr-new.yaml

```diff
@@ -0,0 +1,26 @@
+name: Publish Test Releases
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get changed packages
+        id: changed-packages
+        run: |
+          echo "::set-output name=changed_packages::$(git diff --name-only main ./packages | awk -F'/' '{print "./packages/" $2}' | sort -u | tr '\n' ' ')"
```

Comment:
love a good bash oneliner

Reply:
I *think* it should work 😅 It seemed to work locally, but actions are weird